libnavui-dropin/src/main/res/layout/mapbox_extendable_button_layout.xml

```diff
@@ -12,24 +12,25 @@
         android:id="@+id/container"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:minHeight="@dimen/mapbox_extendableButton_minHeight"
-        android:minWidth="@dimen/mapbox_extendableButton_minWidth"
-        android:paddingStart="@dimen/mapbox_extendableButton_paddingStart">
+        android:layout_gravity="center"
```

Comment:
I don't see the need for `android:layout_gravity="center"`. Work fine even after removing this line.